packages/db/src/config.ts

```diff
@@ -5,11 +5,12 @@ import { Knex } from 'knex';
 import { envUtils } from '@cumulus/common';
 
 export const localStackConnectionEnv = {
+  PG_DATABASE: 'postgres',
   PG_HOST: 'localhost',
-  PG_USER: 'postgres',
   PG_PASSWORD: 'password',
-  PG_DATABASE: 'postgres',
   PG_PORT: '5432',
+  PG_USER: 'postgres',
+  DISABLE_PG_SSL: 'true',
```

Comment:
What is the reasoning for having this default to disabled? I don't disagree, just want to understand.

Reply:
Oh because localstack and we don't want to mess with SSL config locally.

Reply:
Exactly that - setting up SSL on a localstack postgres database is cumbersome.